Demo/EarlGreyExample/EarlGreyContribSwiftTests/GetText.swift

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Example Usage:
+ *
+ *  let element = Element()
+ *
+ *  domainField.performAction(grey_typeText("hello.there"))
+ *             .performAction(grey_getText(element))
+ *
+ *  GREYAssertTrue(element.text != "", reason: "get text failed")
+ */
+func grey_getText(elementCopy: Element) -> GREYActionBlock {
```

Comment:
> FunctionalTests test EarlGrey itself and the new matchers aren't intended to be accepted into EarlGrey yet. If it's fine to just add the experimental matchers directly to core and test them like all the other features, then FunctionalTests makes sense. Do we have a way to mark APIs as experimental? I expect them to change (potentially quite a bit).

FunctionalTests just tests any implemented matchers. The demo project is simply there for setup questions and shouldn't really be looked at for matcher etc. questions. I think we should add it to the FunctionalTests project since we already have Matcher questions / goto examples there. Let's keep this waiting then till `grey_firstElement` is supplanted by `grey_nthElement`.

Thanks a lot for the contributions!